docs/quick-start.md

```diff
@@ -544,113 +581,154 @@ For more details, check out the [VMAlert documentation](https://docs.victoriamet
 
 ## Access
 
-We need to look at the results of what we got. Up until now, we've just been looking only at the status of the pods. 
+In Kubernetes, services are usually accessed within the cluster using internal addresses like:
+```
+http://[SERVICE].[NAMESPACE].svc:[PORT]
+```
+This works well for internal communication between services\pods. 
+But if you want to access a service from outside the cluster (for example, from your browser), it's more complicated.
+There is `kubectl port-forward`, which works well for debugging but isn’t practical for production use.
+It also doesn’t support authentication, routing, enhanced load balancing. 
 
-#### VMAuth
+To access services in a more reliable and secure way, we’ll use:
+- [VMAuth](https://docs.victoriametrics.com/operator/resources/vmauth/) - receives HTTP requests, authenticate and routes them to the correct VictoriaMetrics resources.
+- [VMUser](https://docs.victoriametrics.com/operator/resources/vmuser/) - defines a username and a password, and tells `VMAuth` which requests should go where.
 
-Let's expose our components with [`vmauth`](https://docs.victoriametrics.com/operator/resources/vmauth).
+Before you begin:
+You’ll need an [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) running in your cluster.
+This guide doesn’t include steps for setting up Ingress Controller because it depends on which Kubernetes distribution you're using (like EKS, GKE, or Minikube).
+Also, to keep things simple, we’re skipping HTTPS/TLS in this example — but in production, always use HTTPS to keep your traffic secure.
```

Comment:
Shall we link to how to do this https://docs.victoriametrics.com/victoriametrics/vmauth/#tls-termination-proxy ?

Reply:
Added, but it doesn't explain how to configure TLS with vmauth in Kubernetes.
In the future, it would be helpful to have a Kubernetes-specific version of the guide.